plugin/src/main/java/com/denizenscript/denizen/utilities/command/NPCCommandHandler.java

```diff
@@ -367,7 +366,7 @@ else if (args.hasValueFlag("anchor")) {
                     return;
                 }
             }
-            Messaging.sendError(sender, "The NPC does not have the specified anchor!");
+            Messaging.sendError(sender, "NPC " + npc.getName() + " does not have the anchor '" + args.getFlag("anchor") + "'!");

```

Comment:
Test this with NPCs that have colored names and make sure it compensates appropriate (same for other lines you added name to)

Reply:
I forgot about the possibility of color coding, so I will reset it after the name.